src/components/Xmp.react.js

```diff
@@ -6,6 +6,7 @@ import {omit} from 'ramda';
 const Xmp = (props) => {
     return (
         <xmp
+            data-dash-is-loading={props.loading_state.is_loading}
```

Comment:
Is it possible for `loading_state` to be undefined | null?

Reply:
`loading_state` is a default prop, so no.